src/constant.py

```diff
@@ -6,3 +6,7 @@
 PRINT_BOARD_PLACE_MARBLE = "PRINT_BOARD_PLACE_MARBLE"
 # Key for printing rotation inputs to rotate a quarter
 PRINT_BOARD_ROTATE = "PRINT_BOARD_ROTATE"
+
```

Comment:
it is good to have a constant file but you mix UI, inner constants. You should divide them in several constant files because they are not the same meaning